<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="10" passed="8">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="33936" started-at="2018-04-20T04:37:08Z" finished-at="2018-04-20T04:37:42Z">
    <groups>
    </groups>
    <test name="10DayWeatherForeCast" duration-ms="33936" started-at="2018-04-20T04:37:08Z" finished-at="2018-04-20T04:37:42Z">
      <class name="com.testcases.Day2">
        <test-method status="FAIL" signature="test()[pri:0, instance:com.testcases.Day2@1d057a39]" name="test" duration-ms="2803" started-at="2018-04-20T00:37:11Z" finished-at="2018-04-20T00:37:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test Failed
	at org.testng.Assert.fail(Assert.java:94)
	at com.testcases.Day2.test(Day2.java:100)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day2 -->
      <class name="com.testcases.Day3">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day3@26be92ad]" name="test" duration-ms="2624" started-at="2018-04-20T00:37:14Z" finished-at="2018-04-20T00:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day3 -->
      <class name="com.testcases.Day4">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day4@4c70fda8]" name="test" duration-ms="2677" started-at="2018-04-20T00:37:17Z" finished-at="2018-04-20T00:37:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day4 -->
      <class name="com.testcases.Day5">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day5@224edc67]" name="test" duration-ms="2617" started-at="2018-04-20T00:37:19Z" finished-at="2018-04-20T00:37:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day5 -->
      <class name="com.testcases.Day6">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day6@14acaea5]" name="test" duration-ms="2565" started-at="2018-04-20T00:37:22Z" finished-at="2018-04-20T00:37:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day6 -->
      <class name="com.testcases.Day7">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day7@46d56d67]" name="test" duration-ms="2528" started-at="2018-04-20T00:37:25Z" finished-at="2018-04-20T00:37:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day7 -->
      <class name="com.testcases.Day10">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day10@28d25987]" name="test" duration-ms="2567" started-at="2018-04-20T00:37:32Z" finished-at="2018-04-20T00:37:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day10 -->
      <class name="com.testcases.Day8">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day8@d8355a8]" name="test" duration-ms="2620" started-at="2018-04-20T00:37:27Z" finished-at="2018-04-20T00:37:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day8 -->
      <class name="com.testcases.Day9">
        <test-method status="PASS" signature="test()[pri:0, instance:com.testcases.Day9@59fa1d9b]" name="test" duration-ms="2647" started-at="2018-04-20T00:37:30Z" finished-at="2018-04-20T00:37:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.testcases.Day9 -->
      <class name="com.testcases.Day1">
        <test-method status="FAIL" signature="test()[pri:0, instance:com.testcases.Day1@475530b9]" name="test" duration-ms="3217" started-at="2018-04-20T00:37:08Z" finished-at="2018-04-20T00:37:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test Failed
	at org.testng.Assert.fail(Assert.java:94)
	at com.testcases.Day1.test(Day1.java:100)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day1@475530b9]" name="closeBrowser" is-config="true" duration-ms="690" started-at="2018-04-20T00:37:35Z" finished-at="2018-04-20T00:37:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day10@28d25987]" name="closeBrowser" is-config="true" duration-ms="706" started-at="2018-04-20T00:37:36Z" finished-at="2018-04-20T00:37:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day2@1d057a39]" name="closeBrowser" is-config="true" duration-ms="707" started-at="2018-04-20T00:37:36Z" finished-at="2018-04-20T00:37:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day3@26be92ad]" name="closeBrowser" is-config="true" duration-ms="707" started-at="2018-04-20T00:37:37Z" finished-at="2018-04-20T00:37:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day4@4c70fda8]" name="closeBrowser" is-config="true" duration-ms="707" started-at="2018-04-20T00:37:38Z" finished-at="2018-04-20T00:37:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day5@224edc67]" name="closeBrowser" is-config="true" duration-ms="707" started-at="2018-04-20T00:37:38Z" finished-at="2018-04-20T00:37:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day6@14acaea5]" name="closeBrowser" is-config="true" duration-ms="708" started-at="2018-04-20T00:37:39Z" finished-at="2018-04-20T00:37:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day7@46d56d67]" name="closeBrowser" is-config="true" duration-ms="705" started-at="2018-04-20T00:37:40Z" finished-at="2018-04-20T00:37:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day8@d8355a8]" name="closeBrowser" is-config="true" duration-ms="688" started-at="2018-04-20T00:37:41Z" finished-at="2018-04-20T00:37:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.testcases.Day9@59fa1d9b]" name="closeBrowser" is-config="true" duration-ms="715" started-at="2018-04-20T00:37:41Z" finished-at="2018-04-20T00:37:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.testcases.Day1 -->
    </test> <!-- 10DayWeatherForeCast -->
  </suite> <!-- Suite -->
</testng-results>
